bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSHandler.java

```diff
@@ -293,35 +303,12 @@ public void onError(String error) {
 
     public void postUpdate(String channelId, State state) {
         updateState(channelId, state);
```

Comment:
Please a dd a test here is the channel is linked. If not, no need to update its state.

Reply:
are you sure this is required?

Reply:
Do you really think this is needed?

Reply:
It is of course not mandatory, just better ;) This will avoid running useless code.